### ISSUE TITLE
Conductor UI - Task Logs tab doesn't scroll down, pagination buttons are unreachable

### ISSUE THREAD

**Issue by marcocrasso** [Netflix/conductor#2877]
**Describe the bug**
When task logs are more than those that fit in the browser window, the UI doesn't scroll down and the user cannot access the pagination buttons. 

**Details**
Conductor version: [main](https://github.com/Netflix/conductor/commit/a0e731432ab90798de14fa4257712ff8ea25e73b) branch (at the moment of writing this)
Persistence implementation: Elasticsearch v6 and v7 
Queue implementation: Any
Lock: noop_lock
Workflow definition: Any
Task definition: Any
Event handler definition: N/A

**To Reproduce**

1. start postgres v6 with:
```
docker run --name conductor_es -d --rm -p 9400:9200 -p 9500:9300 -v $(pwd)/docker/opt/es/data:/usr/share/elasticsearch/data -e "discovery.type=single-node" -e "xpack.security.enabled=false" docker.elastic.co/elasticsearch/elasticsearch:6.8.15
```
2. you need a server configured with taskLogResultLimit bigger than 10 (default)
```
git fetch
git checkout main
git pull
```

3. Edit/add into src/main/resources/application.properties next lines:

```
conductor.indexing.enabled=true
conductor.elasticsearch.url=http://localhost:9400/
conductor.elasticsearch.taskLogResultLimit=30
conductor.elasticsearch.clusterHealthColor=yellow
```

5. ./gradlew clean server
6. start UI (`cd ui ; yarn start`)
7. Generate TaskLog. For doing this, there are options. The obvious alternative is to define a wf, with at least one task, and implement one worker that includes TaskLog(s) in the response. That was my case, and I accidentally found this bug.  Optionally, you can use the REST API to populate the logs of an executed task "/task-resource/log", I would recommend this which doesn't require to deploy a new "logger" worker.

**Expected behavior**
UI User should be able to scroll down and access pagination buttons

**Screenshots**
![image](https://user-images.githubusercontent.com/73197513/160156629-a81054fd-e654-4065-a0f1-0ece69fc5aec.png)

**Additional context**
Chrome Version 99.0.4844.83 (Official Build) (x86_64)

**Comment by peterlau** [comment]
Thanks for the bug report. This has been reproduced and will be patched in the next minor release. (eta next week)

**Comment by peterlau** [comment]
Fixed in 3.7.1
